docs/notebooks/wasserstein_barycenters_gmms.ipynb

```diff
@@ -0,0 +1,497 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    x_init = b_cost.means_and_covs_to_x(x_init_means, x_init_covs)
</pre><p>I wonder if it would not make sense to move those helper functions means_and_covs_to_x and vice-versa <em>outside</em> of the BuresCost class, but keep them in the cost.py file, at the root level. That way it's easier to call them without having an object instantiated.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/89/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>OK, I did that. I also added there the custom segmented needed. I would suggest to move all of those in a "helper function file" under tot/tools/gaussian_mixture.</p><p> </p><p>Unless we discuss a different organization on order to have the possibility of custom segmenters and initializers for barycenters of other cost functions as well.</p>